script/scaffold/gather_info.py

```diff
@@ -86,6 +88,22 @@ def gather_new_integration(determine_auth: bool) -> Info:
                 ]
             ],
         },
+        "iot_class": {
+            "prompt": (
+                f"""How will your integration gather data?
+
+Valid values are {', '.join(SUPPORTED_IOT_CLASSES)}
+
+More info @ https://www.home-assistant.io/blog/2016/02/12/classifying-the-internet-of-things/#classifiers
```

Comment:
Calculated is missing in the blog post, it is included the developer documentation.
```suggestion
More info @ https://developers.home-assistant.io/docs/creating_integration_manifest#iot-class
```